### ISSUE TITLE
Consider adding documentation on how to expose container ports to devices outside the Docker host

### ISSUE THREAD

**Issue by toolness** [docker/for-win#3028]
It's often very useful to be able to connect to a container service from the LAN that a Docker host is on.  For example, web developers often have to test their apps on cell phones and tablets, which is most easily accomplished by visiting the app from a device on the local network.

There is surprisingly little documentation on the web about how to do this in Docker for Windows; the vast majority of web searches yield results for accessing a container's services from the Docker host _only_, while other results are about allowing Windows Firewall to permit drive sharing.

After much tinkering, I discovered a solution to this, which I just [posted to Stack Overflow](https://stackoverflow.com/a/53614068/2422398):

> **TLDR:** If you have Windows Firewall enabled, make sure that there is an exception for "vpnkit" on private networks.
>
> For my particular case, I discovered that Windows Firewall was blocking my connection when I tried visiting my container's published port from another machine on my local network, because disabling it made everything work.
>
> However, I didn't want to disable the firewall entirely just so I could access my container's service. This begged the question of which "app" was listening on behalf of my container's service. After finding [another SO thread](https://stackoverflow.com/a/48199/2422398) that taught me to use `netstat -a -b` to discover the apps behind the listening sockets on my machine, I learned that it was `vpnkit.exe`, which already had an entry in my Windows Firewall settings: but "private networks" was disabled on it, and once I enabled it, I was able to visit my container's service from another machine without having to completely disable the firewall.

So the solution is actually pretty simple, but it's _extremely_ hard to find.  I personally think it's a common enough use case to warrant including it in the documentation for Docker for Windows, e.g. in the [logs and troubleshooting](https://docs.docker.com/docker-for-windows/troubleshoot/) section, but I'm obviously biased. 😉
